src/about.js

```diff
@@ -1,18 +1,22 @@
 import Gtk from "gi://Gtk";
+import { gettext as _ } from 'gettext';
 
 export default function About({ application, version }) {
   // https://gjs-docs.gnome.org/gtk30~3.24.8/gtk.aboutdialog
   const dialog = new Gtk.AboutDialog({
+    application,
     authors: ["Sonny Piers https://sonny.re", "Aral Balkan https://ar.al/"],
-    comments: "Commit message editor",
+    comments: _("Commit message editor"),
     copyright:
-      "Copyright 2020-2021 Sonny Piers\nCopyright 2018-2020 Aral balkan",
+      "© 2020-2021 Sonny Piers\n© 2018-2020 Aral balkan",
```

Comment:
@pervoj Should this be translatable ?

Reply:
In the case where only names are involved, I don't think it needs to be translatable.